include/f_core/radio/c_lora_link.h

```diff
@@ -0,0 +1,92 @@
+#ifndef C_LORA_LINK_H
+#define C_LORA_LINK_H
+
+#include <array>
+
+#include <zephyr/kernel.h>
+
+#include <f_core/radio/c_lora.h>
+
+#include "f_core/messaging/c_msgq_message_port.h"
+
+static constexpr uint16_t RADIO_MAX_FRAME_SIZE = UINT8_MAX;
+
+// Launch standardized LoRa packet.
+typedef struct __attribute__((packed)) {
+    uint16_t Port;
+    uint8_t Size;
+    uint8_t Payload[256 - sizeof(uint16_t)];
```

Comment:
I think this should be RADIO_MAX_FRAME_SIZE - sizeof(uint16_t). rn i believe we're carrying around an extra byte and when we use sizeof(Payload) for calcs we're off by one since